dali/test/python/test_operator_element_extract.py

```diff
@@ -142,12 +142,21 @@ def test_element_extract_layout():
     for device in ["cpu", "gpu"]:
         yield check_element_extract, [4, 3, 3], "FXY", [0, 1, 2, 3, 3, 2, 1, 0], device
 
-@raises(RuntimeError)
-def check_element_extract_raises(shape, layout, element_map, dev):
-    check_element_extract(shape, layout, element_map, dev)
-
 def test_raises():
-    for shape, layout in [([4], "F"), ([6, 1], "XF"), ([8, 10, 3], "HWC")]:
-        yield check_element_extract_raises, shape, layout, [1, 3], "cpu"
-    yield check_element_extract_raises, [6, 1], "FX", [10], "cpu"
-    yield check_element_extract_raises, [6, 1], "FX", [-5], "cpu"
+    with assert_raises(RuntimeError,
+                       glob="Input must have at least two dimensions*"
+                       "outermost for sequence*at least one for data"):
+        check_element_extract([4], "F", [1, 3], "cpu")
+
+    for shape, layout in [([6, 1], "XF"), ([8, 10, 3], "HWC")]:
+        with assert_raises(RuntimeError,
+                           glob="Input layout must describe a sequence*start with 'F', got*instead"):
+            check_element_extract(shape, layout, [1, 3], "cpu")
+
+    with assert_raises(RuntimeError,
+                       glob="Index*10*from *element_map* out of bounds for*sequence length*6*"):
```

Comment:
I'm not about this pattern. wildcards inside doesn't provide any flexibility from my point of view, so you can put the whole message here as well.

Reply:
Now I'm just worried, that we will want to fix the `'`, \` and `"` around the variables.

Reply:
But then we can adjust the test, it would be easily detectable.

Reply:
It is unclear for me what we want to leave undefined and what to be strict about. `*` just for `"` or ` blurs the pattern from my point of view.